src/types/Good.ts

```diff
@@ -1,4 +1,5 @@
 export interface Good {
+  [x: string]: unknown;
```

Comment:
why do you need to use it? make it work without this line

Reply:
Oh, right, I think I needed it at one point to check how the different code works and forgot to remove it, thanks :)